researchstack-mpower/src/main/java/org/sagebionetworks/research/mpower/researchstack/ResearchStackTaskLauncher.java

```diff
@@ -33,7 +36,8 @@ public ResearchStackTaskLauncher(@NonNull MpTaskFactory mpTaskFactory) {
     public void launchTask(@NonNull Activity activity, @NonNull String taskIdentifier,
             @Nullable UUID taskRunUUID, int requestCode) {
         Task task = mpTaskFactory.createTask(activity, taskIdentifier);
-
-        activity.startActivityForResult(ActiveTaskActivity.newIntent(activity, task), requestCode);
+        Intent intent = IntentFactory.INSTANCE.newTaskIntent(
+                activity, MpViewTaskActivity.class, task);

```

Comment:
Why did you switch this?

Reply:
different styles when launching into MpViewTaskActivity